test/indexer/test_hca_indexer.py

```diff
@@ -840,6 +840,17 @@ def test_cell_line_sample(self):
                 self.assertEqual(one(contents['cell_suspensions'])['organ'], ['blood (child_cell_line)'])
                 self.assertEqual(one(contents['cell_suspensions'])['organ_part'], [config.null_keyword])
 
+    def test_files_content_description(self):
+        self._index_canned_bundle(('ffac201f-4b1c-4455-bd58-19c1a9e863b4', '2019-10-09T170735.528600Z'))
+        hits = self._get_all_hits()
+        for hit in hits:
+            contents = hit['_source']['contents']
+            entity_type, aggregate = config.parse_es_index_name(hit['_index'])
+            if entity_type in ('files', 'bundles') or not aggregate:
```

Comment:
Still wrong, I think. The `content_description` should be in documents for all ~entities~ entity types, and for both, contributions and aggregates. Maybe you're missing an aggregator/accumulator?